src/clj/harja/kyselyt/kulut.sql

```diff
@@ -295,3 +295,36 @@ LIMIT 1;
 
 -- name: tarkista-kohdistuksen-yhteensopivuus
 SELECT * FROM tarkista_t_tr_ti_yhteensopivuus(:tehtava-id::INTEGER, :tehtavaryhma-id::INTEGER, :toimenpideinstanssi-id::INTEGER);
+
+-- name: hae-toteutuneet-kustannukset-analytiikalle
+SELECT k.id                        AS "kulu-id",
+       k.laskun_numero             AS "laskun-tunniste",
+       k.lisatieto                 AS "kulun-kuvaus",
+       k.poistettu                 AS "poistettu",
+       k.koontilaskun_kuukausi     AS "kulun-ajankohta_koontilaskun-vuosi",-- Huom.
+       k.koontilaskun_kuukausi     AS "kulun-ajankohta_koontilaskun-kuukausi",-- Huom.
```

Comment:
Näyttää vähän oudolta omaan silmään. miltä kuulostaisi jos tämän palauttaisi vain kerran, joko koontilaskun kuukautena tai vaikka "koontilaskun-vuosi-ja-kuukausi", joka sitten puretaan clj-puolella kuukaudeksi ja vuodeksi?

Reply:
Erittäin hyvä pointti. Vaihdoin.